server/helpers/youtube-dl/youtube-dl-wrapper.ts

```diff
@@ -71,13 +116,17 @@ class YoutubeDLWrapper {
     const youtubeDL = await YoutubeDLCLI.safeGet()
 
     const files = await youtubeDL.getSubs({ url: this.url, format: 'vtt', processOptions: { cwd } })
-    if (!files) return []
+    if (!files) {
```

Comment:
Please don't update code you did you modify :)

Reply:
Okay, will revert.

Reply:
Reverted